File: Java Pattern Programs/Symbol Patterns/Pattern 49.java
class Heart_Pattern
{
   public static void main (String args[])
   {
      int i;
      int j;
      int n = 5; //size

   /upper part/

      for (i = n / 2; i <= n; i += 2)
      {
         for (j = 1; j < n - i; j += 2)
         {
            System.out.print(" ");
         }

         for (j = 1; j <= i; j++)
         {
            System.out.print("*");
         }

         for (j = 1; j <= n - i; j++)
         {
            System.out.print(" ");
         }

         for (j = 1; j <= i; j++)
         {
            System.out.print("*");
         }

            System.out.println();
      }

   /lower part/

      for (i = n; i >= 1; i--)
      {
         for (j = i; j < n; j++)
         {
            System.out.print(" ");
         }

         for (j = 1; j <= (i * 2) - 1; j++)
         {
            System.out.print("*");
         }

            System.out.println();
      }

   }
}

File: PythonPatternPrograms/Symbol Patterns/Pattern 49.py
n = 5  # size

# Upper Part
for x in range(n // 2, n + 1, +2):
    for y in range(1, n - x, +2):
        print(" ", end=" ")
    for y in range(1, x + 1):
        print("*", end=" ")
    for y in range(1, n - x + 1):
        print(" ", end=" ")
    for y in range(1, x + 1):
        print("*", end=" ")
    print()

# Lower Part
for x in range(n, 0, -1):
    for y in range(x, n):
        print(" ", end=" ")
    for y in range(1, (x * 2)):
        print("*", end=" ")
    print()for x in range(1, 6):
    for y in range(65, 70):
        print(chr(y), end="")
    print()